### ISSUE TITLE
CrashBrowser: Cursor NPE

### ISSUE THREAD

**Issue by moxie0** [WhisperSystems/TextSecure#2589]
`````
java.lang.NullPointerException: Attempt to invoke virtual method 'void android.support.v4.widget.CursorAdapter.changeCursor(android.database.Cursor)' on a null object reference
at org.thoughtcrime.securesms.ConversationFragment.onLoadFinished(ConversationFragment.java:310)
at org.thoughtcrime.securesms.ConversationFragment.onLoadFinished(ConversationFragment.java:52)
at android.support.v4.app.LoaderManagerImpl$LoaderInfo.callOnLoadFinished(LoaderManager.java:427)
at android.support.v4.app.LoaderManagerImpl$LoaderInfo.onLoadComplete(LoaderManager.java:395)
at android.support.v4.content.Loader.deliverResult(Loader.java:104)
at org.thoughtcrime.securesms.util.AbstractCursorLoader.deliverResult(AbstractCursorLoader.java:39)
at org.thoughtcrime.securesms.util.AbstractCursorLoader.deliverResult(AbstractCursorLoader.java:11)
at android.support.v4.content.AsyncTaskLoader.dispatchOnLoadComplete(AsyncTaskLoader.java:223)
at android.support.v4.content.AsyncTaskLoader$LoadTask.onPostExecute(AsyncTaskLoader.java:61)
at android.support.v4.content.ModernAsyncTask.finish(ModernAsyncTask.java:461)
at android.support.v4.content.ModernAsyncTask.access$500(ModernAsyncTask.java:47)
at android.support.v4.content.ModernAsyncTask$InternalHandler.handleMessage(ModernAsyncTask.java:474)
at android.os.Handler.dispatchMessage(Handler.java:102)
at android.os.Looper.loop(Looper.java:135)
at android.app.ActivityThread.main(ActivityThread.java:5221)
at java.lang.reflect.Method.invoke(Native Method)
at java.lang.reflect.Method.invoke(Method.java:372)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:899)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:694)
`````

**Comment by mcginty** [comment]
The last time there was a changeCursor at ConversationFragment.java:310 was in v2.4.0. Is this still happening?

**Comment by moxie0** [comment]
Confirmed this is still happening in 2.6.4:

`````
java.lang.NullPointerException: Attempt to invoke virtual method 'void android.support.v4.widget.CursorAdapter.changeCursor(android.database.Cursor)' on a null object reference
at org.thoughtcrime.securesms.ConversationFragment.onLoadFinished(ConversationFragment.java:280)
at org.thoughtcrime.securesms.ConversationFragment.onLoadFinished(ConversationFragment.java:49)
at android.support.v4.app.LoaderManagerImpl$LoaderInfo.callOnLoadFinished(LoaderManager.java:427)
at android.support.v4.app.LoaderManagerImpl$LoaderInfo.onLoadComplete(LoaderManager.java:395)
at android.support.v4.content.Loader.deliverResult(Loader.java:104)
at org.thoughtcrime.securesms.util.AbstractCursorLoader.deliverResult(AbstractCursorLoader.java:39)
at org.thoughtcrime.securesms.util.AbstractCursorLoader.deliverResult(AbstractCursorLoader.java:11)
at android.support.v4.content.AsyncTaskLoader.dispatchOnLoadComplete(AsyncTaskLoader.java:223)
at android.support.v4.content.AsyncTaskLoader$LoadTask.onPostExecute(AsyncTaskLoader.java:61)
at android.support.v4.content.ModernAsyncTask.finish(ModernAsyncTask.java:461)
at android.support.v4.content.ModernAsyncTask.access$500(ModernAsyncTask.java:47)
at android.support.v4.content.ModernAsyncTask$InternalHandler.handleMessage(ModernAsyncTask.java:474)
at android.os.Handler.dispatchMessage(Handler.java:102)
at android.os.Looper.loop(Looper.java:135)
at android.app.ActivityThread.main(ActivityThread.java:5256)
at java.lang.reflect.Method.invoke(Native Method)
at java.lang.reflect.Method.invoke(Method.java:372)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:898)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:693)
`````

**Comment by rhodey** [comment]
resolved by 6d2b87d385067cf52abd04d8481a8042d6f681f0
